src/APP/pages/products2/data.js

```diff
@@ -1,201 +1,282 @@
+      title: "Product Designer",
+      headshot: robert,
+      portfolio: "https://www.linkedin.com/in/robert-okusi",
+    },
+    {
+      id: 6,
+      name: "Temitayo Adeleye",
+      title: "Product Designer",
+      headshot: temitayo,
+      portfolio: "https://www.linkedin.com/in/adeleye-temitayo-1964b5b9",
+    },
+    {
+      id: 7,
+      name: "Belinda Koech",
+      title: "Product Designer",
+      headshot: belinda,
+      portfolio: "https://www.linkedin.com/in/thisisbesh",
+    },
+  ],
+  "Front-End Dev": [
+    {
+      id: 1,
+      name: "Rizwan Hamisi",
+      title: "Frontend Developer",
+      headshot: rizwan,
+      portfolio: "https://www.linkedin.com/in/hamisi-rizwan-2b1a00230",
+    },
+    {
+      id: 2,
+      name: "Sonia Lomo",
+      title: "Frontend Developer",
+      headshot:
+        "https://media.licdn.com/dms/image/C4D03AQHnuqOwyIoyeA/profile-displayphoto-shrink_400_400/0/1600620855636?e=1706745600&v=beta&t=Qsr_9AS7NAB8O7dKRLyT-s6OXwRoR-4Uw1cQDk7cGO0",
+      portfolio: "https://www.linkedin.com/in/sonia-lomo/",
+    },
+    {
+      id: 3,
+      name: "Felix Nyalenda",
+      title: "Frontend Developer",
+      headshot: "https://avatars.githubusercontent.com/u/68052150?v=4",
+      portfolio: "https://www.linkedin.com/in/felix-olali-1652aa231",
+    },
+    {
+      id: 4,
+      name: "David Clinton",
+      title: "Frontend Developer",
+      headshot: dave,
+      portfolio: "https://www.linkedin.com/in/clintondavid46/",
+    },
+    {
+      id: 5,
+      name: "Collins Kasyoki",
+      title: "Software Developer",
+      headshot: "https://avatars.githubusercontent.com/u/40399921?v=4",
+      portfolio: "https://www.linkedin.com/in/collins-thano",
+    },
+    {
+      id: 6,
+      name: "Lawrence Moindi",
+      title: "Frontend Developer",
+      headshot:
+        "https://media.licdn.com/dms/image/D4D03AQEOYTI8_FXF3Q/profile-displayphoto-shrink_100_100/0/1680982692686?e=1707350400&v=beta&t=Vji_sa-dDdCNCNU4v5MAT5BPb8lc1iCFvMI9qGSWVWk",
+      portfolio: "https://www.linkedin.com/in/lawrence-moindi-55b35773",
+    },
+    {
+      id: 7,
+      name: "Catherine Kiiru",
+      title: "Frontend Developer",
+      headshot: catherine,
+      portfolio: "https://www.linkedin.com/in/catherine-kiiru-47b2688b/",
+    },
+    {
+      id: 8,
+      name: "Ian Mugenya",
+      title: "Frontend Developer",
+      headshot: ian,
+      portfolio: "https://www.linkedin.com/in/ian-mugenya/",
+    },
+    {
+      id: 9,
+      name: "Sharon Jebitok",
+      title: "Frontend Developer",
+      headshot: sharon,
+      portfolio: "https://www.linkedin.com/in/sharon-jebitok/",
+    },
+  ],
+  "Back-End Dev": [
+    {
+      id: 4,
+      name: "James Otieno",
+      title: "Backend Developer",
+      headshot: james,
+      portfolio: "https://www.linkedin.com/in/james-oduor/",
+    },
+    {
+      id: 5,
+      name: "Hellen Wainaina",
+      title: "Backend Developer",
+      headshot: emmy,

```

Comment:
This is using someone else's headshot.

Reply:
Thank you for this @collinskasyoki, can you please tell her to send her headshot?